### PR TITLE
Adding blog posts and projects to portfolio

### LINKED ISSUE #5: Blog Page
- [x] Header
- [x] Mexico Trip
- [x] Cliff Jumping almost dying
- [x] 3D printed rocket launch
- [x] Footer

### PR DESCRIPTION
**Pull request by jaysaleh**
This pull request includes the addition of 5 posts in this blog, 2 trips and 3 projects that I have worked on. Files to review: `index.html` and `style.css`. This branch does not include a `script.js` file as it is not necessary for the current functionality.

Screenshots:
https://screenshot.googleplex.com/K6VLjyexZwt.png
https://screenshot.googleplex.com/KhBJ4XG4O4Q
https://screenshot.googleplex.com/S5g4XmD4ntP

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,111 @@
...
+        </div>
+      </div>
+      <hr>
+      <div class="post">
+        <div class="post-text">
+          <h1>Toilet Bowl Cliff Jumping</h1>
+          <p>
+            Following my high school graduation, I went on a weekend trip with my friends to Santa Cruz, California.
+            During our weekend trip, one of our friends had the brilliant idea to go cliff jumping at the 
+            <i><b><a href="https://bit.ly/31jEZYq" target="_blank" class="link">Toilet Bowl.</a></b></i>
```
Comment (billie-cat) [inline portfolio/src/main/webapp/index.html]: nit: the period should not be included in the link

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,18 +1,111 @@
...
+      <div class="post">
+        <div class="post-img">
+          <img src="./images/rocket.jpg" alt="Rocket">
+        </div> 
+        <div class="post-text">
+          <h1>3D Printed Rocket</h1>
+          <p>
+            In high school, I began tinkering with 3D printing and rocketry. At the start of my junior year,
+            working with a classmate and a physics teacher, we were able to set up a 3D printer lab on campus
+            consisting of 15 printers for any one to use. The image on the right is taken off a GoPro our rocketry
```
Comment (billie-cat) [inline portfolio/src/main/webapp/index.html]:
```suggestion
            consisting of 15 printers for anyone to use. The image on the right is taken off a GoPro our rocketry
```

**Review by billie-cat — COMMENTED** [review]
Can you add a more descriptive pull request name?

You can read about good CL descriptions here. Think of the title as the first line of the CL description (if that makes sense).
https://google.github.io/eng-practices/review/developer/cl-descriptions.html

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jaysaleh) [reply portfolio/src/main/webapp/index.html:43]: done

Comment (jaysaleh) [reply portfolio/src/main/webapp/index.html]: done

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (jaysaleh) [reply portfolio/src/main/webapp/index.html]: done

**Review by billie-cat — APPROVED** [review]

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -1,9 +1,176 @@
...
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+* {
+  font-family: Helvetica;
 }
 
-#greeting-container {
-  margin-top: 20px;
+/* Header styling */
```
Comment (f-camp) [inline portfolio/src/main/webapp/style.css]: Please order this file declarations alphabetically.
https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order

**Review by f-camp — COMMENTED** [review]
lgtm save minor formatting comment on css file.

**Review by billie-cat — APPROVED** [review]

File: portfolio/src/main/webapp/style.css (hunk 3 above)
Comment (jaysaleh) [reply portfolio/src/main/webapp/style.css]: done

**Review by f-camp — APPROVED** [review]

**Pull request merged**